web/vital_records/templates/vital_records/_form_progress.html

```diff
@@ -1,8 +1,56 @@
-<ol class="cagov-step-list sm d-none d-md-block d-lg-block">
-    <li data-status="{% if url_name == 'request_name' %}current{% endif %}">Name</li>
-    <li data-status="{% if url_name == 'request_county' %}current{% endif %}">County of birth</li>
-    <li data-status="{% if url_name == 'request_dob' %}current{% endif %}">Date of birth</li>
-    <li data-status="{% if url_name == 'request_parents' %}current{% endif %}">Parents’ names</li>
-    <li data-status="{% if url_name == 'request_order' %}current{% endif %}">Order info</li>
-    <li data-status="{% if url_name == 'request_submit' %}current{% endif %}">Preview & submit</li>
-</ol>
+<ul class="vertical-tracker list-unstyled sm m-t-0 d-none d-md-block d-lg-block">
+    <li class="row {% if url_name == 'request_name' %}current{% endif %}"
+        {% if url_name == 'request_name' %}aria-current="true"{% endif %}>
+        <div class="offset-md-2 col-md-10 pos-rel brd-md-left brd-gray-100 p-x-md">
+            <div class="timeline-dot top-0">
+                <span class="dot-line-inner bg-white brd-gray-100"></span>
+            </div>
+            <p class="small">Name</p>
+        </div>
+    </li>
```

Comment:
Would it make sense to extract this out as an include template?

Reply:
Yeah, I think it does. But I was thinking that would be in a future ticket. 

Once we add the marriage steps, we'll have to figure out how to represent each record type's steps (URLs and step name) and iterate the template over each step. 

Could be part of tomorrow's discussions.